isotope/convert/pkg/graph/script/concurrent_command_test.go

```diff
@@ -33,17 +35,25 @@ func TestConcurrentCommand_UnmarshalJSON(t *testing.T) {
 			nil,
 		},
 		{
-			[]byte(`[{"sleep": "1s"}]`),
+			[]byte(`[{"sleep": {"type": "static", "data": {"time": "1s"}}}]`),
```

Comment:
Define type as real type. Perhaps just int32

Reply:
I don't fully understand what you mean by "type as real type". Do you want me to change the type from string to my own types?

Reply:
yes, your own types.